src/analysis/retail/evoker/preservation/modules/talents/DreamBreath.tsx

```diff
@@ -48,6 +92,81 @@ class DreamBreath extends Analyzer {
     };
   }
 
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS_EVOKER.DREAM_BREATH_TALENT.id} />
+        </b>{' '}
+        is one of your empowered abilities and a very strong HoT. You should aim to use it at
+        Empower 1 in most scenarios, with the rare exception when you desperately need a burst AoE
+        heal. If talented into <SpellLink id={TALENTS_EVOKER.CALL_OF_YSERA_TALENT.id} />, always use{' '}
+        <SpellLink id={TALENTS_EVOKER.VERDANT_EMBRACE_TALENT} /> prior to casting{' '}
+        <SpellLink id={TALENTS_EVOKER.DREAM_BREATH_TALENT.id} />.
+      </p>
+    );
+
+    const entries: BoxRowEntry[] = [];
+    this.casts.forEach((cast) => {
+      let value = QualitativePerformance.Fail;
+      const coyActive =
+        this.selectedCombatant.hasTalent(TALENTS_EVOKER.CALL_OF_YSERA_TALENT) && cast.coyActive;
+      if (coyActive && cast.targetsHit === 5) {
+        value = QualitativePerformance.Good;
+      } else if (coyActive && cast.targetsHit === 4) {
+        value = QualitativePerformance.Ok;
```

Comment:
Is it always considered a bad cast if they use DB without the CoY buff, even if they hit 5 targets?

Reply:
Yes, not using Call of Ysera is inting.